docs/UserGuide.md

```diff
@@ -185,10 +187,12 @@ Examples:
 
 Edits an existing task in the task list, at least one field needs to be edited.
 
-Format: `edit TASK_NUMBER [-n TASK_NAME] [-m MODULE] [-d DATE] [-t TAG_NAME*]`
+Format: `edit TASK_NUMBER [-n TASK_NAME] [-m MODULE] [-d DATE] [-t TAG_NAME]*`
+
+**Note**: Params in `[]` are **optional**, and `-t TAG_NAME` can be used multiple times.
```

Comment:
Do we need this if we already mentioned it above? Maybe have a link or something idk